main.js

```diff
@@ -0,0 +1,7 @@
+var logger
+
+if (typeof global !== 'undefined' && global.console) {
```

Comment:
Ah - this pr makes more sense now. You will end up bundling both the node and browser versions in your nashorn bundle but I guess that's OK.